reventlov/bot.py

```diff
@@ -60,6 +61,25 @@ def help_message(self):
             msg = '{}\n{}'.format(msg, handler_msg)
         return msg.replace('_', '\_')
 
+    @property
+    def disabled_plugins(self):
+        return ','.join(
+            [name for name in self.plugins if self.plugins[name] is None]
+        )
+
+    @property
+    def loaded_plugins(self):
```

Comment:
Why the name of this method isn't `loaded_plugins` to maintain to be more consistent with naming?

The function that list _not loaded_ plugins is named `disabled_plugins`.

Reply:
I guess you mean why this method is not calles `enabled_plugins`, don't you?

Reply:
Yes!